{"uuid":"d81df173-c5fb-470d-8eef-f796d54d043f","name":"logging in with multiple fixtures","historyId":"e9a871b24f4d9b27ca08715501ec8023:d41d8cd98f00b204e9800998ecf8427e","status":"broken","statusDetails":{"message":"cy.allure is not a function","trace":"TypeError: cy.allure is not a function\n    at Context.eval (webpack://cypress_backend_automation/./cypress/e2e/login.cy.js:86:11)\n    at runnable.fn (https://practicetestautomation.com/__cypress/runner/cypress_runner.js:140440:19)\n    at callFn (https://practicetestautomation.com/__cypress/runner/cypress_runner.js:159518:21)\n    at Runnable.run (https://practicetestautomation.com/__cypress/runner/cypress_runner.js:159505:7)\n    at <unknown> (https://practicetestautomation.com/__cypress/runner/cypress_runner.js:165577:30)\n    at PassThroughHandlerContext.finallyHandler (https://practicetestautomation.com/__cypress/runner/cypress_runner.js:4042:23)\n    at PassThroughHandlerContext.tryCatcher (https://practicetestautomation.com/__cypress/runner/cypress_runner.js:1777:23)\n    at Promise._settlePromiseFromHandler (https://practicetestautomation.com/__cypress/runner/cypress_runner.js:1489:31)\n    at Promise._settlePromise (https://practicetestautomation.com/__cypress/runner/cypress_runner.js:1546:18)"},"stage":"finished","steps":[],"attachments":[{"name":"Incorrect Log In -- logging in with multiple fixtures (failed).png","source":"ea09decf-dfb8-44ca-ab7b-80cd6e50c690-attachment.png","type":"image/png"}],"parameters":[],"labels":[{"name":"language","value":"javascript"},{"name":"framework","value":"cypress"},{"name":"parentSuite","value":"Incorrect Log In"},{"name":"host","value":"Dave101101"},{"name":"thread","value":"pid-28596-worker-main"},{"name":"package","value":"cypress.e2e.login.cy.js"}],"links":[],"start":1749621796885,"fullName":"cypress/e2e/login.cy.js#Incorrect Log In logging in with multiple fixtures","testCaseId":"e9a871b24f4d9b27ca08715501ec8023","stop":1749621797960}